src/Chapter1.hs

```diff
@@ -556,7 +563,14 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+        | (c == 'a' || c == 'A') = True
+        | (c == 'e' || c == 'E') = True
+        | (c == 'i' || c == 'I') = True
+        | (c == 'o' || c == 'O') = True
+        | (c == 'u' || c == 'U') = True
```

Comment:
`()` are redundant here. I see how it can be confusing to see when to use them and when not to use, especially in the beginning. But it's usually better to remove redundant brackets for cleaner code :broom: